src/index.ts

```diff
@@ -25,14 +26,17 @@ export class Writr {
         const program = createCommand();
 
         program.storeOptionsAsProperties(true);
-        
+
         program.option("-p, --path <path>", "Path of where the blog, config, and template are located");
         program.option("-o, --output <path>", "Path of where to output the generated blog");
         program.option("-r, --render <list>", "What do you want rendered such as html or json (example --render html,json)");
         program.option("-c, --config <path>", "custom configuration path");
-    
+        program.option("-j, --jekyll <path>", "Jekyll path to be migrated");
```

Comment:
@christianllv - are you thinking that we would have a command for each migration? Is that scalable as we are talking wordpress, medium, and ghostjs? Would it make more sense to have the command be `-m` or `--migrate` and then the next command be type and params?

```bash
writr --migrate jekyll <path> 
```

Reply:
Totally agree.

My suggestion is writer --migrate jekyll <originPath> <destinationPath>

What do you think?

Reply:
I am good with that

Reply:
Updated, please check